opennms-webapp/src/main/webapp/WEB-INF/applicationContext-spring-security.xml

```diff
@@ -279,12 +279,12 @@
                 password-parameter="j_password"
                 login-processing-url="/j_spring_security_check"
                 authentication-failure-url="/login.jsp?login_error=1"
+                authentication-failure-handler-ref="failureHandler"
+                authentication-success-handler-ref="successHandler"
```

Comment:
Guessing I'm gonna want to backport this to 2020? The spring security changes go back to there.

Reply:
actually, maybe we want to backport this whole thing, seems like it would apply